docs_src/OVMS/quick_pipelinerun.md

```diff
@@ -0,0 +1,136 @@
+        ./download_sample_videos.sh 1920 1080 15;
+        cd ..;
+        ```
+
+    !!! Note
+        Only AVC encoded files are supported.
+
+2. After the video files are downloaded to the **sample-media** folder, start the camera simulator:
+    ```bash
+    make run-camera-simulator
+    ```
+
+3. Wait for few seconds, and then check if the camera-simulator containers are running:
+    ```bash
+    docker ps --format 'table{{.Image}}\t{{.Status}}\t{{.Names}}'
+    ```
+
+    !!! Success
+        Your output is as follows:
+    
+        | IMAGE                                              | STATUS                   | NAMES             |
+        | -------------------------------------------------- | ------------------------ |-------------------|
+        | openvino/ubuntu20_data_runtime:2021.4.2            | Up 11 seconds            | camera-simulator0 |
+        | aler9/rtsp-simple-server                           | Up 13 seconds            | camera-simulator  |
+    
+    
+        !!! Note
+            There could be multiple containers with the image "openvino/ubuntu20_data_runtime:2021.4.2", depending on the number of sample-media video files provided.
+    
+    !!! Failure
+        If all the Docker* containers are not visible, then review the console output for errors. Sometimes dependencies fail to resolve. Address obvious issues and retry.
+
+### Run Instance Segmentation
+
+There are several pipeline profiles to chose from. Use the `make list-profiles` to see the different pipeline options. In this example, the `instance_segmentation` pipeline profile will be used. 
+
+1. Use the following command to run instance segmentation using OVMS on core.
+
+    ```bash
+    PIPELINE_PROFILE="instance_segmentation" RENDER_MODE=1 sudo -E ./run.sh --workload ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
+    ```
+
+2. Check the status of the pipeline.
+   
+    ```bash
+    docker ps --format 'table{{.Image}}\t{{.Status}}\t{{.Names}}' -a
+    ```
+    !!! Success
+        Here is a sample output:
+
+        | IMAGE                                              | STATUS                       | NAMES         |
+        | -------------------------------------------------- | ---------------------------- |---------------|
+        | openvino/model_server-gpu:latest                   | Up 59 seconds                | ovms-server0 |
+
+    !!! Failure
+   
+        If you do not see above Docker container(s), review the console output for errors. Sometimes dependencies fail to resolve and must be run again. Address obvious issues and try again repeating the above steps. Here are couple debugging tips:
+   
+        1. check the docker logs using following command to see if there is an issue with the container
+   
+            ```bash
+            docker logs <containerName>
+            ```
+        2. check ovms log in automated-self-checkout/results/r0.jsonl
+
+3. Check the output in the `results` directory.
```

Comment:
missing indent

Reply:
this actually continues the previous numbering, not the one in the note. :)

Reply:
oh interesting. Maybe we should consider changing the second list to a. b. or 1.1 and 1.2 to help avoid confusion